clustintime/clustintime.py

```diff
@@ -164,6 +165,6 @@ def clustintime(
             prefix=prefix,
         )
 
-    if save_maps == True:
+    if save_maps:
```

Comment:
Same discussion as before.
If save_maps is truthy, this check will always pass. If we want it to be explicitly "True", we need to check for `save_maps is True:"... etc.